AppCenterDistribute/AppCenterDistributeTests/MSDistributeTests.m

```diff
@@ -1461,12 +1461,15 @@ - (void)testDeleteUpdateTokenAfterReinstall {
 - (void)testWithoutNetwork {
 
   // If
-  id reachabilityMock = OCMClassMock([MS_Reachability class]);
-  OCMStub([reachabilityMock reachabilityForInternetConnection]).andReturn(reachabilityMock);
-  OCMStub([reachabilityMock currentReachabilityStatus]).andReturn(NotReachable);
+  self.currentNetworkStatus = NotReachable;
   id distributeMock = OCMPartialMock(self.sut);
   OCMReject([distributeMock buildTokenRequestURLWithAppSecret:OCMOCK_ANY releaseHash:kMSTestReleaseHash isTesterApp:false]);
 
+  // Reject submitting calls.
+  id ingestionCallMock = OCMPartialMock([MSIngestionCall alloc]);
```

Comment:
That's the third time I see this block. Maybe a small helper method?

Reply:
I guess this is OK for mocking code in tests. We need to be consistent at least.